sdk/version/version_base.go

```diff
@@ -9,6 +9,6 @@ var (
 	CgoEnabled bool
 
 	Version           = "1.8.0"
-	VersionPrerelease = "dev"
+	VersionPrerelease = "rc"
```

Comment:
```suggestion
	VersionPrerelease = "rc2"
```
It needs a number, can you double check that this is right?